### PR TITLE
Added modal component

### PR DESCRIPTION
**Pull request by ar-Ryan**
added html and css files for modal. reviews appreciated.

### CODE REVIEW

File: Components/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.modal-box{
+    display: flex;
+    flex-direction: column;
+    background-color: rgb(228, 228, 228);
```
Comment (saifudinbadani) [inline Components/modal/modal.css:4]: if bg colour is being re-used maybe use variable.

**Review by saifudinbadani — APPROVED** [review]
Otherwise, looks good.

File: Components/modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
+.modal-box{
+    display: flex;
+    flex-direction: column;
+    background-color: rgb(228, 228, 228);
+    padding: 1.25rem;
+    width: 25rem;
+    border: 1px solid grey;
+    border-radius: 4px;
```
Comment (ark2002) [inline Components/modal/modal.css]: i think there is a typo for the color gray here

**Review by ark2002 — APPROVED** [review]
rest looks good

File: Components/modal/modal.css (hunk 2 above)
Comment (ar-Ryan) [reply Components/modal/modal.css]: It works with "grey" with my editor. But i'll still make sure its correct, thanks!

File: Components/modal/modal.css (hunk 1 above)
Comment (ar-Ryan) [reply Components/modal/modal.css:4]: I made these codes separately from the main file, I'll try to implement this in the downloadable CSS sheet.

**Pull request merged**